docs/README.md

```diff
@@ -1042,6 +1045,49 @@ The resulting staging repository will be available in the output named `staging-
           nexus-password: "${{ secrets.NEXUS_PASSWORD }}"
 ```
 
+### nexus-create-tag
+
+Creates a tag on Nexus with a specific name. The resulting tag will be available in the output named `tag-name`.
```

Comment:
I don't see any output being set, probably because it doesn't make sense to have one if it's the same value provided as input

Reply:
I agree, I don't think an out put is needed here